### ISSUE TITLE
unclear text?

### ISSUE THREAD

**Issue by neslekkim** [ArthurHub/HTML-Renderer#86]
I'm using html-renderer to just render some text in my wpf-application, and are finding that some text is getting kinda foggy?, ie, a bit out of focus?

**Comment by Spuffynism** [comment]
Could this be related? https://github.com/ArthurHub/HTML-Renderer/commit/4f3f77e4e8300ece97f6979d7c52a3c09fb4c953

**Comment by AistisT** [comment]
It is, that causes the problem, but the problem that is supposedly fixed by that is a real one. But making everything blurry is a poor way to fix it.

**Comment by neslekkim** [comment]
Yes, When I removed the code that was introduced in commit 4f3f77e4e8300ece97f6979d7c52a3c09fb4c953 then the blurryness dissapeared, what was that supposed to fix?

**Comment by Spuffynism** [comment]
@neslekkim see https://github.com/ArthurHub/HTML-Renderer/pull/62 for more details

**Comment by Licshee** [comment]
sorry for this, gonna revert
